lib/srv/alpnproxy/kube.go

```diff
@@ -81,7 +81,8 @@ type KubeMiddleware struct {
 	// headless controls whether proxy is working in headless login mode.
 	headless bool
 
-	logger logrus.FieldLogger
+	logger  logrus.FieldLogger
+	context context.Context
```

Comment:
Same here. A more specific name could be useful when deciding if this is the right context for a given flow.